### PR TITLE
a

### PR DESCRIPTION
**Pull request by PassiHD2004**
* Create gradle.yml

* fix: fix github workflow

* fix: fix warnings
feat: add modmenu as dependency

---------

### CODE REVIEW

**Review by RoootTheFox — CHANGES_REQUESTED** [review]
I'm not a fan of adding mod menu as a hard dependency as it isn't required.

**Comment by RoootTheFox** [comment]
assert this.client != null;

**Comment by PassiHD2004** [comment]
whats with that?

**Comment by RoootTheFox** [comment]
useless

**Comment by PassiHD2004** [comment]
intellij was throwing errors, so yeah "could produce NullPointerException"

**Comment by RoootTheFox** [comment]
just ignore those, make IntelliJ shut the ever living crap up

**Comment by PassiHD2004** [comment]
everything fine now?

File: src/main/java/net/foxes4life/foxclient/screen/pause/PauseScreen.java [hunk 1]
```diff
@@ -90,7 +91,7 @@ public void init() {
         }
 
 
-        buttonWidget.active = this.client.isIntegratedServerRunning() && !this.client.getServer().isRemote();
+        buttonWidget.active = this.client.isIntegratedServerRunning() && this.client.getServer().isRemote();
```
Comment (RoootTheFox) [inline src/main/java/net/foxes4life/foxclient/screen/pause/PauseScreen.java:94]: whoops?

File: src/main/java/net/foxes4life/foxclient/screen/pause/PauseScreen.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
 import net.minecraft.util.Util;
 
 import java.lang.reflect.InvocationTargetException;
+import java.util.Objects;
```
Comment (RoootTheFox) [inline src/main/java/net/foxes4life/foxclient/screen/pause/PauseScreen.java:20]: LMAO

**Comment by RoootTheFox** [comment]
this PR was an experience...
lgtm

**Pull request merged**